### PR TITLE
feat(BUILD-11536): disable systemd-resolved on ubuntu18.04

### PR DESCRIPTION
**Pull request by zakhark**
### Description

Disable systemd-resolved on ubuntu 18.04 because it would still use that service over dnsmasq

### CODE REVIEW

File: tasks/disable-systemd-resolved.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- name: Disable systemd-resolved
+  systemd:
+    name: systemd-resolved
+    enabled: no
```
Comment (jasonwalsh) [inline tasks/disable-systemd-resolved.yml:4]:
```diff
- enabled: no
+ enabled: false
```

File: tasks/main.yml [hunk 2]
```diff
@@ -69,3 +69,9 @@
   changed_when: false
   ignore_errors: true
   when: result.stat.exists
+
+- name: Disable systemd-resolved if needed
+  include_tasks: disable-systemd-resolved.yml
+  when:
+    - ansible_distribution == "Ubuntu"
+    - ansible_distribution_major_version is version("18", "==")
```
Comment (jasonwalsh) [inline tasks/main.yml:77]: Does this only apply to Ubuntu 18.x releases? What about 19.x and 20.x?

File: tasks/disable-systemd-resolved.yml (hunk 1 above)
Comment (zakhark) [reply tasks/disable-systemd-resolved.yml]: Fixed

File: tasks/main.yml (hunk 2 above)
Comment (zakhark) [reply tasks/main.yml]: Updated to ">="

**Pull request merged**